refinery/file_store/tasks.py

```diff
@@ -154,8 +171,8 @@ def import_file(uuid, refresh=False, file_size=0):
 
                     # check if we have a sane value for file size
                     if remote_file_size > 0:
-                        percent_done = local_file_size * 100. / \
-                                       remote_file_size
+                        percent_done = \
+                            local_file_size * 100. / remote_file_size
```

Comment:
Probably but I don't know enough about this to comment. I simply re-arranged existing code for readability.